### PR TITLE
Added backup.sh script

### PR DESCRIPTION
**Pull request by dlopez-akalam**

### CODE REVIEW

File: bin/backup.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+# Ensure directories exist
+ensure_directory ${BACKUPS_DIR_SITE}
+ensure_directory ${BACKUPS_DIR_DB}
+
+# Do the backups
+STAMP=`date +%y%m%d-%H%M%S`;
+CURRENT_DIR=$(basename `pwd`)
+drush sql-dump --gzip > ${BACKUPS_DIR_DB}/${BACKUP_PREFIX}-db-${STAMP}.sql-gz;
```
Comment (rsanzante) [inline bin/backup.sh:65]: I would add a check to make sure this is a drupal folder.

This how the fix permissions script does the trick:
```
if [ -z "${drupal_path}" ] || [ ! -d "${drupal_path}/sites" ] || [ ! -f "${drupal_path}/core/modules/system/system.module" ] && [ ! -f "${drupal_path}/modules/system/system.module" ]; then
  printf "**********************************************\n"
  printf "* Error: Please provide a valid Drupal path. *\n"
  printf "**********************************************\n"
  print_help
  exit 1
fi
```
See  https://www.drupal.org/node/244924#script-based-on-guidelines-given-above

File: bin/backup.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#  on the users home.
+#  This script can be called directly
+#  or automated by cron, as it do a
+#  cleanup for oldest backups.
+#
+#             WARNING!
+#  Execute from the project root
+#######################################
+
+set -x
```
Comment (rsanzante) [inline bin/backup.sh]: This is the debug flag. I would use -e instead, so it stops on error. Debug can be enabled from the command line if needed.

**Comment by dlopez-akalam** [comment]
@rsanzante Your comments were achieved. Could you review it again?

**Comment by rsanzante** [comment]
Merged!

**Pull request merged**